.gitignore

```diff
@@ -4,7 +4,7 @@ build/
 !gradle/wrapper/gradle-wrapper.jar
 !**/src/main/**/build/
 !**/src/test/**/build/
-.env
+/src/.env
```

Comment:
Er det nødvendigt med /src/.env? Bør .env ikke være nok for at ignorere filtypen?